.circleci/config.yml

```diff
@@ -18,77 +23,63 @@ parameters:
     default: "dev:alpha"
 
 jobs:
-  # This job is an example of an integration testing job.
-  # This job should execute a command from your orb and verify
-  # the output is as expected, otherwise the job should fail.
-  #
-  # Rename this job and add more as needed.
-  #
-  integration-test-1:
-    executor: orb-tools/ubuntu
+  integration-tests:
```

Comment:
What about keeping some comments explaining that this is just an example of the job? Without that, it takes some time to understand why we set params explicitly here, and what is it overall...

Reply:
Or rename `integration-tests` to `example-job` or so

Reply:
Actually, these are tests that run every time you check in